packages/documentation/docs/controls/_textarea-field_styleguide.mdx

```diff
@@ -0,0 +1,62 @@
+The Textarea component allows users to input multi-line text. It can be used in various forms where longer text input is required.
+
+![Textarea overview](https://www.figma.com/design/wEptRgAezDU1z80Cn3eZ0o/iX-Pattern-Illustrations?node-id=3814-1128&t=DtCmoFcLwhf7ke3S-4)
+1. Label
+2. Required field indicator (optional)
```

Comment:
Normally, we never specified in anatomy whether something is optional or not. If desired, we would need to consistently maintain this in all Markdowns.